evalml/tests/component_tests/test_one_hot_encoder.py

```diff
@@ -494,12 +514,19 @@ def test_ohe_column_names_unique():
     df = pd.DataFrame({"A": ["x_y"], "A_x": ["y"]})
     df_transformed = OneHotEncoder().fit_transform(df)
     assert set(df_transformed.columns) == {"A_x_y", "A_x_y_1"}
-    df = pd.DataFrame({"A": ["x_y", "z"], "A_x": ["y_1", "y"], "A_x_y": ["1", "y"]})
+
+    df = pd.DataFrame({"A": ["x_y", "z", "z"], "A_x": ["y", "a", "a", ], "A_x_y": ["1", "y", "y"]})
```

Comment:
Why did we add a third row here? Was the sorting of the majority class not deterministic?

Reply:
I believe it is, but it's dependent on the impl of `value_counts`--I wanted to *not* rely on that :d

Reply:
Great! Just wanted to verify.